README.md

```diff
@@ -174,7 +174,8 @@ SELECT
     Duration / 1000000 AS duration,
     Timestamp AS startTime,
     arrayMap(key -> map('key', key, 'value', SpanAttributes[key]), mapKeys(SpanAttributes)) AS tags,
-    arrayMap(key -> map('key', key, 'value', ResourceAttributes[key]), mapKeys(ResourceAttributes)) AS serviceTags
+    arrayMap(key -> map('key', key, 'value', ResourceAttributes[key]), mapKeys(ResourceAttributes)) AS serviceTags,
+    if(StatusCode = 'STATUS_CODE_ERROR', 2, 0) AS statusCode
```

Comment:
Update to `Error`, verify indentation

Reply:
By "verify indentation" do mean that it should be indented with using two spaces instead of four?

Reply:
The indentation looked a little off when viewing it in the diff, just wanted to verify that it was correct